src/components/studio-footer/StudioFooter.jsx

```diff
@@ -134,13 +135,15 @@ const StudioFooter = ({
           <FormattedMessage {...messages.trademarkMessage} />
           <Hyperlink className="ml-1" destination="https://www.edx.org">edX Inc</Hyperlink>.
           <ActionRow.Spacer />
-          <Hyperlink destination="https://open.edx.org" className="float-right">
-            <Image
-              width="120px"
-              alt="Powered by Open edX"
-              src="https://logos.openedx.org/open-edx-logo-tag.png"
-            />
-          </Hyperlink>
+          <PluginSlot id="studio_footer_logo_slot">
```

Comment:
Would you be able to follow the pattern of making a new component for the slot and including a README with screenshots?

You can see that pattern used in https://github.com/openedx/frontend-component-header/tree/master/src/plugin-slots

Reply:
Part of the pattern I linked is moving the `PluginSlot`s out into their own components that live next to the README files with examples. Would you be able to do that for this one?